src/test/js/reset_password_test.js

```diff
@@ -21,7 +21,7 @@ BeforeSuite(async (I) => {
     adminEmail = 'admin.' + randomData.getRandomEmailAddress();
     citizenEmail = 'citizen.' + randomData.getRandomEmailAddress();
     otherCitizenEmail = 'other.' + randomData.getRandomEmailAddress();
-    plusCitizenEmail = 'plus.' + "extra+" + randomData.getRandomEmailAddress();
+    plusCitizenEmail = 'plus.' + 'extra+' + randomData.getRandomEmailAddress();
```

Comment:
`plus.extra${randomData.getRandomEmailAddress()}`
also would be nice to have another character to separate extra from random data:
`plus.extra.${randomData.getRandomEmailAddress()}`